### PR TITLE
Ft refactor wirebot 158583916

### PR DESCRIPTION
**Pull request by andela-venogwe**
#### What does this PR do?
Refactor the wirebot codebase.

#### Any background context you want to add?
- jsdoc style documentation
- use the standard style for auto linting

#### Description of Task to be completed?
1. Refactor codebase.
2. Document existing and new functions

#### How should this be manually tested
- Create, configure and install a slack app by following the readme.
- Go to slack and type command `/report`.
- go through the incident creation process

#### What are the relevant pivotal tracker stories?
#158583916 Refactor Wirebot

### CODE REVIEW

**Comment by andela-venogwe** [comment]
@bmuthoga cant connect to slack on my system, sorry for not communicating via slack

File: modules/services.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ * @param {Object} payload the incident created on the wire api
+ *
+ * @returns {Promise} a promise containing the notification response
+ */
+async function notifyPAndChannels (payload) {
+  try {
+    const { reporter: [{ location: { country } }] } = payload
+    const channel = PNC_CHANNELS.split(',').find(
+      value => value.toLowerCase().includes(country.toLowerCase())
+    ).replace(`-${country.toLowerCase()}`, '')
```
Comment (kimobrian) [inline modules/services.js]: Not sure of the current slack names for these channels. Will have to confirm this when we have the first production deployment.

File: wirebot.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      logger.error('port is already in use')
+      return process.exit(1)
+    default:
+      return logger.error(error.message)
+  }
+}
+
+app.use(helmet())
+app.options(setHeadersMiddleware)
+app.use(bodyParser.urlencoded({ extended: false }))
```
Comment (kimobrian) [inline wirebot.js]: Will need to update to `express.urlencoded...` and do away with body-parser

File: modules/services.js (hunk 1 above)
Comment (andela-venogwe) [reply modules/services.js]: sure, as a rule they should conform to prefixing it with country names if they can help it.

**Pull request merged**